src/core_plugins/kibana/index.js

```diff
@@ -56,6 +56,7 @@ module.exports = function (kibana) {
           return {
             kbnDefaultAppId: serverConfig.get('kibana.defaultAppId'),
             tilemapsConfig: {
+              my_app_version: serverConfig.get('pkg.version'),
```

Comment:
We shouldn't pass variables in snake_case. Just inject the `kbnVersion` on the client side, it's already available in angular.